src/core/lib/surface/call.h

```diff
@@ -71,6 +71,8 @@ void grpc_call_internal_unref(grpc_call* call);
 #define GRPC_CALL_INTERNAL_UNREF(call, reason) grpc_call_internal_unref(call)
 #endif
 
+gpr_arena* grpc_call_get_arena(grpc_call* call);
```

Comment:
I'm not sure that just adding new function removes the need for the grpc_call_get_arena function. Since it's an internal test, let me chat with you offline about it.